app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivity.kt

```diff
@@ -94,8 +94,9 @@ class AdministratorControlsActivity :
       supportFragmentManager.findFragmentById(
         R.id.administrator_controls_fragment_multipane_placeholder
       )
-    /* when fragment is ProfileEditFragment then we should switch fragment to ProfileListFragment or else
-       we just end the activity as then we are having ProfileListFragment on top. */
+    /* If we have current fragment ProfileListFragment then we should end the activity on back press
+       but if we have ProfileEditFragment as current fragment then we should inflate profileListFragment
+       which is done by the handleOnBackPressed method. */
```

Comment:
```suggestion
    /*
     * If the current fragment is ProfileListFragment then the activity should end on back press.
     * If it's instead ProfileEditFragment then profileListFragment should be inflated via
     * handleOnBackPressed.
     */
```

Some reformatting and language changes. This might also need re-wrapping to more tightly fit the 100 character limit (I didn't check since it's a bit difficult to via a web text box).